Source/DafnyLanguageServer.Test/Synchronization/DiagnosticsTest.cs

```diff
@@ -181,7 +181,7 @@ decreases y
     }
 
     [TestMethod]
-    public async Task OpeningDocumentWithMultipleVerificationErrorsReportsDiagnosticsWithAllVerificationErrors() {
+    public async Task OpeningDocumentWithMultipleVerificationErrorsReportsDiagnosticsWithAllVerificationErrorsAndRelatedInformation() {
```

Comment:
(idle thought) Boy this makes me wish we could declare tests with arbitrary strings as some test frameworks support. :) I think there's an xUnit plugin example out there that shows how to customize the test discovery to support that...